packages/x-charts/src/PieChart/PieArcPlot.tsx

```diff
@@ -118,6 +125,28 @@ function PieArcPlot(props: PieArcPlotProps) {
           {...slotProps?.pieArc}
         />
       ))}
+      {/* Render the focus indicator last, so it can align nicely over all arcs */}
+      {focusedItem && (
+        <Arc
+          key={focusedItem.dataIndex}
+          startAngle={focusedItem.startAngle}
+          endAngle={focusedItem.endAngle}
+          paddingAngle={focusedItem.paddingAngle}
+          innerRadius={focusedItem.innerRadius}
+          color={focusedItem.color}
+          outerRadius={focusedItem.outerRadius}
+          cornerRadius={focusedItem.cornerRadius}
+          skipAnimation
+          stroke={(theme.vars ?? theme).palette.text.primary}
+          id={id}
+          className={'MuiPieArc-focusIndicator'}
```

Comment:
We can add `focusIndicator` to `pieArcClasses` and use it here as such:


```suggestion
          className={pieArcClasses.focusIndicator}
```